ppx/ppx_deriving_rpcty.cppo.ml

```diff
@@ -124,12 +124,14 @@ module Typ_of = struct
         [ Vb.mk (pvar typ_of_lid) (polymorphize (expr_of_typ manifest))]
       | Ptype_record labels, _ ->
         let fields =
+          let one_field = match labels with [_] -> true | _ -> false in
+          let over = if one_field then None else Some ([%expr s]) in
```

Comment:
Shouldn't it be `let over s = ...`? Or it does not matter because `expr` is rewritten?

Reply:
Yes, looking at the bare documentation that seems correct. I still feel uncomfortable about that, what about something like:
```ocaml
let fset = [%expr fun v s ->
    let over = if one_field then None else Some ([%expr s]) in
    [%e record [fname, [%expr v]] ?over:()]]
in
```

Reply:
I've updated the code with your suggestion.